src/MessageReader.js

```diff
@@ -59,11 +59,11 @@ class StandardTypeReader {
       return String.fromCharCode.apply(null, codePoints);
     }
```

Comment:
How does your approach compare to this `apply` method? Is it fast enough that we can remove the `apply` method altogether?

Reply:
I did some benchmarks and I think we should actually use a TextDecoder, with a simple fallback of string concatenation. This should give us the memory advantages on browsers we care about (Chrome).
100 very large strings (50k chars, similar to what we see in our largest messages)
TextDecoder: ~ 33ms
String concatentation: ~60ms
string.join(""): ~115ms
fromCharCode: ~220ms